Mathlib/Probability/Cdf.lean

```diff
@@ -86,11 +87,26 @@ lemma measure_cdf [IsProbabilityMeasure μ] : (cdf μ).measure = μ := by
   refine Measure.ext_of_Iic (cdf μ).measure μ (fun a ↦ ?_)
   rw [StieltjesFunction.measure_Iic _ (tendsto_cdf_atBot μ), sub_zero, ofReal_cdf]
 
+end ExplicitMeasureArg
+
+lemma cdf_measure_stieltjesFunction (f : StieltjesFunction) (hf0 : Tendsto f atBot (𝓝 0))
+    (hf1 : Tendsto f atTop (𝓝 1)) :
+    cdf f.measure = f := by
+  refine (cdf f.measure).eq_of_measure_of_tendsto_atBot f ?_ (tendsto_cdf_atBot _) hf0
+  have h_prob : IsProbabilityMeasure f.measure :=
+    ⟨by rw [f.measure_univ hf0 hf1, sub_zero, ENNReal.ofReal_one]⟩
+  exact measure_cdf f.measure
+
 end ProbabilityTheory
 
 open ProbabilityTheory
 
 /-- If two real probability distributions have the same cdf, they are equal. -/
-lemma MeasureTheory.Measure.ext_of_cdf (μ ν : Measure ℝ) [IsProbabilityMeasure μ]
+lemma MeasureTheory.Measure.eq_of_cdf (μ ν : Measure ℝ) [IsProbabilityMeasure μ]
     [IsProbabilityMeasure ν] (h : cdf μ = cdf ν) : μ = ν := by
   rw [← measure_cdf μ, ← measure_cdf ν, h]
+
+lemma MeasureTheory.Measure.eq_iff_cdf (μ ν : Measure ℝ) [IsProbabilityMeasure μ]
+    [IsProbabilityMeasure ν] :
+    μ = ν ↔ cdf μ = cdf ν :=
+⟨fun h ↦ by rw [h], MeasureTheory.Measure.eq_of_cdf μ ν⟩
```

Comment:
This looks like it would be a good `simp` lemma if stated the other way round. How about (untested):
```suggestion
@[simp] lemma MeasureTheory.Measure.cdf_eq_iff (μ ν : Measure ℝ) [IsProbabilityMeasure μ]
    [IsProbabilityMeasure ν] :
    cdf μ = cdf ν ↔ μ = ν :=
⟨MeasureTheory.Measure.eq_of_cdf μ ν, fun h ↦ by rw [h]⟩
```